distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/Client.java

```diff
@@ -65,58 +70,59 @@ public void runTest(String[] args) throws Exception {
         Query standingAllQuery = new Query(NsiliConstants.NSIL_ALL_VIEW,
                 "NSIL_CARD.identifier like '%'");
         if (SHOULD_TEST_STANDING_QUERY_MGR) {
-            nsiliClient.testStandingQueryMgr(orb, rootPOA, standingAllQuery);
+            nsiliClient.testStandingQueryMgr(rootPOA, standingAllQuery);
         }
 
         // CatalogMgr
-        Query query = new Query(NsiliConstants.NSIL_ALL_VIEW, "NSIL_CARD.identifier like '%' and NSIL_CARD.sourceLibrary = 'test'");
+        Query query = new Query(NsiliConstants.NSIL_ALL_VIEW,
+                "NSIL_CARD.identifier like '%' and NSIL_CARD.sourceLibrary = 'test'");
         int hitCount = nsiliClient.getHitCount(query);
         if (hitCount > 0) {
             DAG[] results = nsiliClient.submit_query(query);
             if (results != null && results.length > 0) {
                 nsiliClient.processAndPrintResults(results, SHOULD_DOWNLOAD_PRODUCT);
 
                 //OrderMgr
-                nsiliClient.order(orb, rootPOA, results);
+                nsiliClient.order(orb, results);
 
                 //ProductMgr
-                System.out.println("-----------------------");
+                LOGGER.info("-----------------------");
                 try {
-                    nsiliClient.testProductMgr(orb, rootPOA, results);
+                    nsiliClient.testProductMgr(orb, results);
                 } catch (Exception e) {
-                    System.out.println("Unable to test ProductMgr: "
-                            + NsilCorbaExceptionUtil.getExceptionDetails(e));
+                    LOGGER.info("Unable to test ProductMgr: {}",
```

Comment:
NsilCorbaExceptionUtil.getExceptionDetails(e) extracts the meaningful information from the CORBA exceptions  (which is sometimes bloated with meaningless information)